web/components/ui/Content/Content.tsx

```diff
@@ -305,6 +303,24 @@ export const Content: FC = () => {
           handleClose={() => setShowFollowModal(false)}
         />
       </Modal>
+      {showChatModal && isChatVisible && (
+        <ChatModal
+          messages={messages}
+          currentUser={currentUser}
+          handleClose={() => setShowChatModal(false)}
+        />
+      )}
+      {isChatVisible && (
+        <Button
+          id="mobile-chat-button"
+          type="primary"
+          onClick={() => setShowChatModal(true)}
+          className={styles.floatingMobileChatModalButton}
+          style={{ zIndex: 99 }}
+        >
+          Chat <MessageFilled style={{ overflow: 'auto', position: 'relative', top: '2px' }} />
```

Comment:
Not sure how `top: 2px` is working here with `relative`, but using transform translate is preferred. `transform: translateY(2px);`